### ISSUE TITLE
Uncaught TypeError: _d2.default.tip is not a function

### ISSUE THREAD

**Issue by eevah** [Caged/d3-tip#145]
Hi!

I installed d3-tip (using npm) but when I tried the sample code:

    var tip = d3.tip()
      .html("hi there");

I receive error Uncaught TypeError: _d2.default.tip is not a function

I also tried simply including "index.js" (d3-tip) file and requiring it in my code, but I get the same error. I don't think it has something to do with the file requirement though, because other libraries in my project work fine. Do you have any tips on how to resolve this issue? Am I missing something?

Help is appreciated!

Thanks

**Comment by aaronfullerton** [comment]
+1 Did you ever come to a conclusion @eevah?

**Comment by oluckyman** [comment]
I found a workaround:
```javascript
import d3Tip from 'd3-tip';
// ...
// ...
const tip = d3Tip().html(function(d) { return d; });
d3.select(this.refs.svg).call(tip);
```

**Comment by kuy** [comment]
@oluckyman Nice tips. I put the exported function to `d3.tip` for compatibility.

```es6
import d3 from 'd3';
import tip from 'd3-tip';

d3.tip = tip;

// ...

tip = d3.tip()
    .attr(...)
   .offset(...)
   // ...
```

**Comment by vdh** [comment]
Fixed by #162
